src/analytics.js

```diff
@@ -244,6 +245,155 @@ const Analytics = {
 
 		return await Analytics.client;
 	},
+	audiences: [],
+	updateAudiences: async () => {
```

Comment:
is this actually more like `fetchAudiences`? The naming now looks like "Add audiences to someone", but I thikn maybe it means "Update (fetch) the audiences"?

Reply:
Not quite, it looks at the available data (attributes, metrics and endpoint) then figures what audiences the current page session belongs to. So it's not really "fetching" anything, at least in the HTTP sense.

Ultimately it sets the `audiences` array on the line above which can then be retrieved so I think "update" makes the most sense.